### ISSUE TITLE
Output panel's "switch to output action" has no effect when hidden

### ISSUE THREAD

**Issue by mjbvz** [microsoft/vscode#162007]
Testing #161802

Very similar to #162004

1. Open the output panel
2. Right click on panel controls to hide the `switch to output`
3. Now use the `...` to try running `switch to output` 

![Screen Shot 2022-09-27 at 9 05 20 AM](https://user-images.githubusercontent.com/12821956/192577870-5e659dec-bd90-40c5-b975-0c8c786a6059.png)

**Bug**
Nothing happens

**Comment by jrieken** [comment]
@sandy081 am I right in assuming that "switch to output" isn't a submenu but a fake action that renders other actions?

**Comment by jrieken** [comment]
Similar to https://github.com/microsoft/vscode/issues/162004 I can add some prevent-to-hide band aid but there might be debt here

**Comment by sandy081** [comment]
Yes. It implements its own action item here - https://github.com/microsoft/vscode/blob/eb99a782ff563ca3f6ed400ab700e3ab41f2243b/src/vs/workbench/contrib/output/browser/outputView.ts#L275
